properties/config.py

```diff
@@ -425,9 +424,9 @@ class LuxCoreConfig(PropertyGroup):
     sampler: EnumProperty(name="Sampler", items=samplers, default="SOBOL")
     
     samplers_gpu = [
-        ("SOBOL", "Sobol", "Best suited sampler for the GPU. " + SOBOL_DESC, 0),
-        ("RANDOM", "Random", RANDOM_DESC, 1),
-    ]
+        ("SOBOL", "Sobol", "Best suited sampler for the GPU. " + SOBOL_DESC, 0),]
+        #("RANDOM", "Random", RANDOM_DESC, 1),
+    #]
```

Comment:
Why was this removed? I think it's sometimes used by the BCD denoiser.

Also why is the code commented out rather than deleted?